pkg/users/users.js

```diff
@@ -153,18 +158,13 @@ function AccountsPage() {
     } else if (path.length === 1) {
         return (
             <AccountDetails accounts={accountsInfo} groups={groupsExtraInfo}
-                            current_user={current_user_info?.name} user={path[0]} shells={shells} />
+                current_user={current_user_info?.name} user={path[0]} shells={shells} />
         );
     } else return null;
 }
 
-async function getLogins() {
-    let lastlog = "";
-    try {
-        lastlog = await cockpit.spawn(["lastlog"], { environ: ["LC_ALL=C"] });
-    } catch (err) {
-        console.warn("Unexpected error when getting last login information", err);
-    }
+async function getLogins(LastLogPath) {
```

Comment:
Hmm, now I would say that `getLogins` can figure out itself which command to call. It could even do `cockpit.shell("if test -e ...; then lastlog2 else lastlog; fi")`.

Reply:
yeah, that makes sense I will change it